test/jdk/java/net/httpclient/offline/DelegatingHttpClient.java

```diff
@@ -112,4 +112,34 @@ public <T> HttpResponse<T> send(HttpRequest request,
               HttpResponse.PushPromiseHandler<T> pushPromiseHandler) {
         return client.sendAsync(request, responseBodyHandler, pushPromiseHandler);
     }
+
+    @Override
+    public boolean isTerminated() {
+        return client.isTerminated();
+    }
+
+    @Override
+    public void shutdown() {
+        client.shutdown();
+    }
+
+    @Override
+    public void shutdownNow() {
+        client.shutdownNow();
+    }
+
+    @Override
+    public boolean awaitTermination(Duration duration) throws InterruptedException {
+        return client.awaitTermination(duration);
+    }
+
+    @Override
+    public void close() { client.close(); }
+
+    /**
+     * Invokes the default implementation of HttpClient::close
+     * Can be closed by subclasses if necessary.
```

Comment:
I think this is a typo in the javadoc and I suspect you meant "Can be overridden by subclasses if necessary."